src/main/java/io/phasetwo/keycloak/magic/auth/MagicLinkAuthenticator.java

```diff
@@ -143,6 +149,17 @@ private boolean isActionTokenPersistent(AuthenticationFlowContext context, boole
     return is(context, ACTION_TOKEN_PERSISTENT_CONFIG_PROPERTY, defaultValue);
   }
 
+  private OptionalInt getActionTokenLifeSpan(
+      AuthenticationFlowContext context, String defaultValue) {
+    String lifespan = get(context, ACTION_TOKEN_LIFE_SPAN, defaultValue);
+
+    if ("".equals(lifespan)) {
+      return OptionalInt.empty();
+    }
+
+    return OptionalInt.of(Integer.parseInt(lifespan));
```

Comment:
Should catch a `NumberFormatException` here, as the string doesn't get validated

Reply:
True, I didn't validate that... there are two approaches here: Forcing an unknown error for Keycloak and preventing login, or going with the default settings and logging an error.

I'm more inclined to use the default configuration. What do you think?